components/schema-migrator/migrations/provisioner/202203171600_operation_alter_last_error.up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE operation

```

Comment:
If you change default values in migration script - you should also update them in provisioner/assets/database/provisioner.sql file

Reply:
added `NOT NULL` in provisioner.sql for last error fields..